<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>CSS Selectors</title>
    <link rel="stylesheet" href="./style.css" />
    <title>My Resume</title>
</head>
<body>
    <h1>Surabhi Seth</h1>
    <hr>
    <h2>PROFILE SUMMARY</h2>
        <p>
            A results-driven professional with expertise in Systems Biology, Text Mining, Machine Learning, and Transcriptomics Analysis.
            Proficient in writing, revising, and reviewing scientific reports, analytical protocols, data analysis reports, specifications, and
            controlled documents. Possesses strong analytical and problem-solving abilities with a proven capacity to work
            independently and within multidisciplinary, team-oriented environments. Demonstrates excellence in communication,
            collaboration, and stakeholder engagement
        </p>
    <hr>
    <h2>CONTACT</h2>
        <p>
            +917521998275<br>
            surabhismo@gmail.com<br>
            <a href = "https://www.linkedin.com/in/surabhi-seth-b2623b228/">Linkedin profile surabhi seth</a><br>
            <a href = "https://github.com/suroseth">github surabhi seth</a><br>
        </p>
    <hr>
    <h2>EDUCATION</h2>
        <p>
        <ul>
            <li><h3>Doctor of Philosophy (PhD)</h3>
            <p>
                Bioinformatics[Thesis writing stage]<br>
                Academy of Scientific and Innovative Research, Faculty of Biological Sciences.<br>
                [CSIR-IGIB]<br>
            </p>
            </li>
            <li><h3>Master's Degree</h3>
            <p>
                Zoology (Specialization in Molecular and Human Genetics)<br>
                Banaras Hindu University<br>
                CGPA: 8.14<br>
                Completed: 2019<br>                
            </p>
            </li>
            <li><h3>Bachelor's Degree</h3>
            <p>
                B.Sc. hons. Zoology<br>
                Banaras Hindu University<br>
                CGPA: 8.33<br>
                Completed: 2017<br>                
            </p>
            </li>            
        </ul> 
        </p>       
    <hr>
    <h2>PROJECTS</h2>
    <p>
        <ul>
            <li><h3>Prediction Model for Type 2 Diabetes Associated Pancreatic Cancer</h3>
                <ul>
                    <li>Conducted text processing and information extraction using pubmed.mineR, Tm, and NLTK.</li>
                    <li>Conducted differential expression (Micro-array and RNA-Seq) and co-expression analysis (WGCNA), pathway simulation (BioNSi).</li>
                    <li>Experience in handling a wide range of ten ML algorithms spanning Linear Classifiers, Tree-Based Methods, Gradient Boosting Methods, Naive Bayes Classifiers, and different algorithm combinations.</li>
                    <li>Constructed database using Streamlit.</li>
                </ul>
            </li>
            <li><h3>Text Mining for Solutions to Antimicrobial Resistance</h3>
                <ul>
                    <li>Designed gene features from literature text mining and transcriptomics data.</li>
                    <li>Visualized data using R and developed a web interface using PHP, HTML, and CSS.</li>
                </ul>
            </li>     
            <li><h3>Disease Comorbidity in Tuberculosis and Type 2 Diabetes Patients</h3>
                <ul>
                    <li>Analyzed microarray data using limma package.</li>
                    <li>Constructed mRNA-miRNA networks and simulated using BioNSi (Cytoscape plugin).</li>
                </ul>
            </li>
            <li><h3>Reverse Vaccinology for Mucormycosis</h3>
                <ul>
                    <li>Created a computational pipeline using tools for adhesin prediction, subcellular localization, and immune simulation.</li>
                    <li>Predicted antigenicity, homology, allergenicity, and T/B cell epitopes.</li>
                </ul>
            </li>
            <li><h3>Phytochemicals Targeting Breast Cancer through Text Mining</h3>
                <ul>
                    <li>Extracted abstracts using easypubmed and performed NLP with pubmed.mineR.</li>
                    <li>Enhanced functionality of pubmed.mineR with a custom co-occurrence analysis function.</li>
                </ul>
            </li>
            <li><h3>Pan-India SARS-CoV-2 Sero-Epidemiological Survey</h3>
                <ul>
                    <li>Performed antibody detection using Electro-Chemiluminescence Immunoassay (ECLIA).</li>
                </ul>
            </li>
            <li><h3>Linking Type 2 Diabetes and Psoriasis via Perturbed Pathways</h3>
                <ul>
                    <li>Conducted literature mining, and DE analysis, to construct a comorbidity network and pathway simulation with BioNSi.</li>
                </ul>
            </li>
            <li><h3>Analysed trends and gaps through text mining and generative AI</h3>
                <ul>
                    <li>Research Trends on Employee Wellness Using Text Mining and Generative AI.</li>
                </ul>
            </li>
            <li><h3>AMR Prediction Using Machine Learning</h3>
                <ul>
                    <li>Built gene-family-specific ML prediction models for AMR detection using gene sequence in ESKAPE pathogens.</li>
                </ul>
            </li>
            <li><h3>Endothelial Dysfunction and Homocysteine</h3>
                <ul>
                    <li>Analyzed microarray data to identify pathways impacted by elevated homocysteine in endothelial cells.</li>
                </ul>
            </li>                                                                        
        </ul>
    </p>        
    <hr>
    <h2>SKILLS</h2>
        <ul>
            <li><h3>Technical Skills:</h3>
                <ul>
                    <li>Advanced Machine Learning & Artificial Intelligence</li>
                    <li>Comprehensive Transcriptomics Analysis & Interpretation</li>
                    <li>Text Mining & Natural Language Processing</li>
                    <li>Interactive Data Visualization</li>
                    <li>Complex Network Construction & Analysis</li>
                    <li>Systems Biology & Pathway Simulation</li>
                    <li>Computational Biology & Bioinformatics</li>
                    <li>Database Architecture & Management</li>
                </ul>
            </li>
            <li><h3>Programming & Development:</h3>
                <ul>
                    <li>Languages: R, Python</li>
                    <li>Frameworks: Web applications using Streamlit, HTML, CSS, PHP, XAMPP</li>
                    <li>Environments: Linux</li>
                </ul>

            </li>

            <li><h3>Bioinformatics Tools:</h3>
                <ul>
                    <li>Network Analysis: Cytoscape, WGCNA, BioNSi, MCODE</li>
                    <li>Genomic Databases: TCGA, GEO, cBioPort</li>
                    <li>Protein Analysis: BUSCA, TargetP, BLASTP, AlgPred, BepiPred, ProtParam</li>
                    <li>Immunoinformatics: TepiTool, VaxiJen, C-ImmSim</li>
                </ul>
            </li>
            <li><h3>Professional Skills:</h3>
                <ul>
                    <li>Scientific Writing & Research Publication</li>
                    <li>Cross-functional Team Collaboration</li>
                    <li>Complex Problem-Solving & Critical Thinking</li>
                    <li>Project Management & Experimental Design</li>
                </ul>
            </li>
        </ul>
    <hr>
    <h2>AWARDS & ACHIEVEMENTS</h2>
    <ul>
        <li>Junior Research Fellowship (All India Rank 64) - CSIR-UGC Fellowship Examination, December 2018.</li>
        <li>Qualified GATE Life Sciences, 2019.</li>
        <li>Participated in Climate Knowledge Hunt Hackathon (2024).</li>
        <li>Poster presented at Indian Cancer Genome Atlas (ICGA) (2024)</li>
        <li>Poster presented at 8th World Cancer Congress – 2024</li>
        <li>Second prize in oral presentation at International Conference on Health and Agricultural Biotechnology (HABIT) 2025.</li>
        <li>Diploma in Kathak Dance with Distinction (2022-2024).</li>
    </ul>
    <hr>
    <h2>PUBLICATIONS</h2>
    <ol>
        
        <li>S. Seth, K. Chaudhary, and S. Ramachandran, “A Machine Learning
        Prediction Model for Type 2 Diabetes Mellitus Associated Pancreatic
        Cancer,” Dec. 05, 2024, Social Science Research Network, Rochester, NY:
        5036037. [pre-print]  <a href="https://papers.ssrn.com/sol3/papers.cfm?abstract_id=5036037">doi:10.2139/ssrn.5036037</a>
        </li>
        <li>Research Article 
        R. Kumar, S. Seth, A. Bhargav, N. Saini, and S. Ramachandran,
        “Computational Identification of Perturbed Pathways in Type 2
        Diabetes Mellitus Patients Reveal Necroptosis and NF-Kb Pathways
        with Potential for Susceptibility to Psoriasis,”. J. Biosci. [Manuscript in communication] <a href="" ></a></li>
        <li> S. Ramachandran, S. Seth, and P. Chaurasia, “The Importance of Food
        Care Services in Healthcare Organizations: Food and Nutrition in
        Hospitals,” in Healthcare Administration and Managerial Training in the
        21st Century, IGI Global Scientific Publishing, 2024, pp. 435–478. doi:
        10.4018/979-8-3693-5523-7.ch014.[Book Chapter]<a href="https://www.igi-global.com/chapter/the-importance-of-food-care-services-in-healthcare-organizations/355365">doi:10.4018/979-8-3693-5523-7.ch014</a></li>
        <li>
        B. Chatterjee, F. Fatima, S. Seth, and S. Sinha Roy, “Moderate Elevation of
        Homocysteine Induces Endothelial Dysfunction through Adaptive UPR
        Activation and Metabolic Rewiring,” Cells, vol. 13, no. 3, p. 214, Jan. 2024,
        doi: 10.3390/cells13030214.<a href="https://pubmed.ncbi.nlm.nih.gov/38334606/">PMID:38334606</a></li>
        <li>
        J. Rani, A. Bhargav, S. Seth, M. Datta, U. Bajpai, and S. Ramachandran,
        “Identification of perturbed pathways rendering susceptibility to
        tuberculosis in type 2 diabetes mellitus patients using BioNSi
        simulation of integrated networks of implicated human genes,” J.
        Biosci., vol. 47, no. 4, p. 69, Dec. 2022, doi: 10.1007/s12038-022-00309-z.<a href="https://pubmed.ncbi.nlm.nih.gov/36476775/">PMID:36476775</a></li>
        <li>
        A. Bhargav, F. Fatima, P. Chaurasia, S. Seth, and S. Ramachandran,
        “Computer-Aided Tools and Resources for Fungal Pathogens: An
        Application of Reverse Vaccinology for Mucormycosis,” Monoclon.
        Antibodies Immunodiagn. Immunother., vol. 41, no. 5, pp. 243–254, Oct.
        2022, doi: 10.1089/mab.2021.0039.<a href="https://pubmed.ncbi.nlm.nih.gov/35939284/">PMID:35939284</a></li>
        <li>
        A. Bhargav, S. Gupta, S. Seth, S. James, F. Fatima, P. Chaurasia, & S.
        Ramachandran, “Knowledgebase of potential multifaceted solutions to
        antimicrobial resistance.” Oct. 2021, doi:
        10.20944/preprints202110.0033.v1.<a href="https://pubmed.ncbi.nlm.nih.gov/36155273/">PMID:36155273</a></li>
        <li>
        S. Naushin, S. Seth, et al., “Insights from a Pan India Sero-
        Epidemiological survey (Phenome-India Cohort) for SARS-CoV2,” eLife,
        vol. 10, p. e66537, doi: 10.7554/eLife.66537.<a href="https://pubmed.ncbi.nlm.nih.gov/33876727 /">PMID:33876727 </a></li>
        
    </ol>
</body>
</html>
